source/sccm/index.rst

```diff
@@ -0,0 +1,73 @@
+SCCM
+====
+
+* Sources link: https://github.com/pluginsGLPI/sccm
+* Download: https://github.com/pluginsGLPI/news/sccm
+
+Requirements for latest version
+-------------------------------
+
+This plugin requires :
```

Comment:
You should add FI plugin as a requirement

Reply:
done